docs/jwst/tweakreg/README.rst

```diff
@@ -100,8 +100,9 @@ If the step parameter ``abs_refcat`` is set to 'GAIADR3', 'GAIADR2' or 'GAIADR1'
 an astrometric reference catalog then gets generated by querying
 a GAIA-based astrometric catalog web service for all astrometrically
 measured sources in the combined field-of-view of the set of input
-images. This catalog gets generated from the catalogs available
-through the GSSS STScI web services at `GSSSCATALOGS`_.
+images. This catalog is generated from the catalogs available
+through the GSSS STScI web services at `GSSSCATALOGS`_ and accounts for proper 
+motion to a given epoch. The default epoch is 2016.0, which results in no proper motion correction.
```

Comment:
Given that the epoch argument is not exposed at the user level, and hence is not settable by the user (at least if they're just running the top-level tweakreg step), I would rephrase this second sentence to:
"The epoch is automatically computed from observation date and time of the input data. If for any reason the epoch cannot be computed from the meta data, it defaults to 2016.0, which results in no proper motion correction."